core/store/eth_client.go

```diff
@@ -12,6 +12,23 @@ import (
 	"github.com/smartcontractkit/chainlink/core/utils"
 )
 
+//go:generate mockgen -package=mocks -destination=../internal/mocks/eth_wrapper_mocks.go github.com/smartcontractkit/chainlink/core/store EthWrapper
+
+// EthWrapper is the interface supplied the EthClient
```

Comment:
Looks like there might be a typo here.  "EthWrapper is the interface supplied by(?) the EthClient"?